.github/workflows/sdk.yaml

```diff
@@ -57,3 +57,47 @@ jobs:
       - run: cargo clippy -- --deny warnings
       - run: cargo fmt
       - run: git diff --exit-code
+  regenerate:
+    # Verifies the generated code has not been tampered with. Or maybe that the
+    # code requires no tampering.
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        rust-version: ['1.81']
+        go-version: ['1.23']
+    steps:
+      - uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/sidekick
+          key: ${{ github.job }}-${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}
+      - uses: actions/checkout@v4
+      - name: Setup Rust ${{ matrix.rust-version }}
+        run: rustup toolchain install ${{ matrix.rust-version }}
+      - name: Display Cargo version
+        run: cargo version
+      - name: Display rustc version
+        run: rustc --version
+      - name: Setup Go ${{ matrix.go-version }}
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ matrix.go-version }}
+          cache-dependency-path: |
+            generator/go.sum
+      - name: Install protoc
+        run: |
+          set -e
+          curl -fSSL -o /tmp/protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v28.3/protoc-28.3-linux-x86_64.zip
+          cd /usr/local
+          sudo unzip -x /tmp/protoc.zip
+          protoc --version
+      - name: Install goimports
```

Comment:
It does't look like you run `goimports` anywhere. Is this being used somewhere?

Reply:
You are right, it is used in `sidekick_test.go` but we are not running that here. Removed.

https://github.com/googleapis/google-cloud-rust/blob/3aa85ba410fd6888effd693f222cfcf047241b42/generator/sidekick/sidekick_test.go#L270